themes/scss/widgets/themes/light.scss

```diff
@@ -11,6 +11,7 @@ $link: rgba(#3AA9E3, 1);
 $linkHover: rgba(#227DBD, 1);
 $lightHighlight: rgba(#000, 0.06);
 $highlighted: rgba(#9DE0AD, 1);
+$selected: rgba(#3AA9E3, 1);
```

Comment:
I wanted to keep them separated since this is within /themes, and that would give us more flexibility when customizing styles.